tests/unittests/config/test_cc_apt_pipelining.py

```diff
@@ -2,27 +2,77 @@
 
 """Tests cc_apt_pipelining handler"""
 
+from pathlib import Path
+
+import pytest
+
 import cloudinit.config.cc_apt_pipelining as cc_apt_pipelining
-from tests.unittests.helpers import CiTestCase, mock
+from cloudinit import util
+from cloudinit.config.schema import (
+    SchemaValidationError,
+    get_schema,
+    validate_cloudconfig_schema,
+)
+from tests.unittests.helpers import (
+    cloud_init_project_dir,
+    mock,
+    skipUnlessJsonSchema,
+)
 
 
-class TestAptPipelining(CiTestCase):
+class TestAptPipelining:
```

Comment:
Generally want to move away for CiTestCase subclassese wherever possible to get the benefits of pytest features/fixtures/etc.